public/app/features/explore/SecondaryActions.test.tsx

```diff
@@ -1,70 +1,77 @@
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import { noop } from 'lodash';
 import React from 'react';
 
 import { SecondaryActions } from './SecondaryActions';
 
-const addQueryRowButtonSelector = '[aria-label="Add row button"]';
-const richHistoryButtonSelector = '[aria-label="Rich history button"]';
-const queryInspectorButtonSelector = '[aria-label="Query inspector button"]';
-
 describe('SecondaryActions', () => {
-  it('should render component two buttons', () => {
-    const wrapper = shallow(
+  it('should render component with three buttons', () => {
+    render(
       <SecondaryActions
         onClickAddQueryRowButton={noop}
         onClickRichHistoryButton={noop}
         onClickQueryInspectorButton={noop}
       />
     );
-    expect(wrapper.find(addQueryRowButtonSelector)).toHaveLength(1);
-    expect(wrapper.find(richHistoryButtonSelector)).toHaveLength(1);
+
+    expect(screen.getByRole('button', { name: /Add row button/i })).toBeInTheDocument();
+    expect(screen.getByRole('button', { name: /Rich history button/i })).toBeInTheDocument();
+    expect(screen.getByRole('button', { name: /Query inspector button/i })).toBeInTheDocument();
   });
 
   it('should not render add row button if addQueryRowButtonHidden=true', () => {
-    const wrapper = shallow(
+    render(
       <SecondaryActions
         addQueryRowButtonHidden={true}
         onClickAddQueryRowButton={noop}
         onClickRichHistoryButton={noop}
         onClickQueryInspectorButton={noop}
       />
     );
-    expect(wrapper.find(addQueryRowButtonSelector)).toHaveLength(0);
-    expect(wrapper.find(richHistoryButtonSelector)).toHaveLength(1);
+
+    expect(screen.queryByRole('button', { name: /Add row button/i })).not.toBeInTheDocument();
+    expect(screen.getByRole('button', { name: /Rich history button/i })).toBeInTheDocument();
+    expect(screen.getByRole('button', { name: /Query inspector button/i })).toBeInTheDocument();
   });
 
   it('should disable add row button if addQueryRowButtonDisabled=true', () => {
-    const wrapper = shallow(
+    render(
       <SecondaryActions
         addQueryRowButtonDisabled={true}
         onClickAddQueryRowButton={noop}
         onClickRichHistoryButton={noop}
         onClickQueryInspectorButton={noop}
       />
     );
-    expect(wrapper.find(addQueryRowButtonSelector).props().disabled).toBe(true);
+
+    expect(screen.getByRole('button', { name: /Add row button/i })).toHaveAttribute('disabled');
```

Comment:
I think this may work as well:

```suggestion
    expect(screen.getByRole('button', { name: /Add row button/i })).toBeDisabled();
```